netci.groovy

```diff
@@ -658,35 +658,52 @@ branchList.each { branchName ->
     }
 }
 
-// Generate the build and test versions for Windows_NT.  When full build/run is supported on a platform, those platforms
+// Generate the build and test versions for Windows_NT and Ubuntu.  When full build/run is supported on a platform, those platforms
 // could be removed from above and then added in below.
-def supportedFullCyclePlatforms = ['Windows_NT']
+def supportedFullCyclePlatforms = ['Windows_NT', 'Ubuntu']
```

Comment:
Renaming here will not affect anything else.. The backend support for Ubuntu14.04 is already [here](https://github.com/dotnet/dotnet-ci/blob/master/jobs/generation/Utilities.groovy#L98)